src/QueryProvider.tsx

```diff
@@ -0,0 +1,20 @@
+import { FC, ReactNode } from 'react'
```

Comment:
Please, do not import FC, it's unreadable to have `FC` as a type. If you see `React.FC` you know that this is a react function component, but when you see just `FC` as a type, you need to check, where this `FC` came from, especially if you are new in this codebase.

Also, if you are importing types, make sure to add `type` prefix to this imports. It is a very good practice, especially if you are using non-type values as a type. This approuch reduces js bundle size.

So, your import will be like this: `import { type ReactNode } from 'react'